fe/src/project/AspNetCore/areas/News/components/NewsItemDetails/NewsItemDetails.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import PropTypes from "prop-types";
+import LazyLoad from "react-lazyload";
+import { CalendarToday, PictureAsPdf, Folder, Attachment } from "@material-ui/icons";
+import LazyLoadConstants from "../../../../../../shared/constants/LazyLoadConstants";
+import ResponsiveImage from "../../../../../../shared/components/Picture/ResponsiveImage";
+import NavigationHelper from "../../../../../../shared/helpers/globals/NavigationHelper";
+import { Hash } from "react-feather"
+import {marked} from "marked";
+import moment from "moment";
+
+const NewsItemDetails = (props) => {
+    const handleDownloadFile = (file) => {
+        NavigationHelper.redirect(file.url)
```

Comment:
**suggestion:** it's a bad idea to use redirect to download file. Use simply the href as in the Files.js module.